PROJECT_GUIDE.md

```diff
@@ -0,0 +1,37 @@
+# Goals/Mission
+
+The mission is to improve developer relations, growth in proficiency, and network for people within LagosJUG and beyond. Feel free to dive into our open-source ecosystem, where contributions are driven by passion and collaboration.
+
+# Project
+A Project at any point in time must have a Lead/Owner and at least 3 team members working to achieve a milestone within its defined scope under the supervision of the Lagos Admin. The Project Owner should document Each Milestone as a user story.
+
+All Lagos JUG projects are Opensource projects, so we prioritize using free Opensource tools. 
+
+Refer to our [Open Source Guides](https://github.com/LagosJavaUserGroup/Documents/tree/main)
+
+# Project Owner
+You Must be a verified member of LagosJUG,
+You own the Project, 
+You articulate and define the Project's goals and objectives. 
+You Prioritize features, requirements, and tasks. 
+You review and approve key project deliverables, ensuring they meet quality standards and align with project objectives. 
+You Work with LagosJUG Admin to secure the necessary resources and budget. 
+You select or delegate the selection of team members to Tech Leads. 
+You provide the Project charter, 
+You define a specific Risk Management Plan for your Project
+
+
+# Team Members
+Must read the Contributors agreement, code of conduct, and the Open source license, issue/pull request. [Open Source Guides](https://github.com/LagosJavaUserGroup/Documents/tree/main)
+Must be a member of LagosJUG
+A team should have at least 1 of the following PM/PO, Developer, and Designer [the designer for FE does the graphics/flow - UI/UX, while for BE does system design - Architect/Lead]. We recommend that separate individuals hold these roles to avoid strain/stress.
```

Comment:
What's
1. PM/PO
2. FE
3. BE
Can these acronyms be spelt out for better understanding to a common man.